Sources/SourceKitLSP/Swift/ExpressionTypeInfo.swift

```diff
@@ -1,103 +0,0 @@
-//===----------------------------------------------------------------------===//
-//
-// This source file is part of the Swift.org open source project
-//
-// Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
-// Licensed under Apache License v2.0 with Runtime Library Exception
-//
-// See https://swift.org/LICENSE.txt for license information
-// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
-//
-//===----------------------------------------------------------------------===//
-
-import Dispatch
-import LanguageServerProtocol
-import SourceKitD
-
-/// A typed expression as returned by sourcekitd's CollectExpressionType.
-///
-/// A detailed description of the structure returned by sourcekitd can be found
-/// here: https://github.com/apple/swift/blob/main/tools/SourceKit/docs/Protocol.md#expression-type
-struct ExpressionTypeInfo {
```

Comment:
Were inlay hints just never hooked up, or are they done some other way now?

Reply:
This was the part of inlay hints that could show type annotations for expressions, not variables. I’m not sure if that ever made it in or if we removed it pretty early on because annotating `1` and `2` in `let x = 1 + 2` to `let x: Int = x: Int + 2: Int` wasn’t really useful.

`VariableTypeInfo` still exists.